### ISSUE TITLE
Implement the CLI `list-languages` and `list-word-types` functionality

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#148]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is already included in the work in #140, but making this to document the changes being made. The `list-languages` functionality of the Scribe-Data CLI will provide a list of languages that the user can get lexeme data from Wikidata with in alphabetical order. The `list-word-types` command would return all possible word types available if no `--language` argument is passed, or if there is one then the results would be filtered for just the available word types of that language (or languages). 

### Contribution

@mhmohona has already implemented the changes for this in #140 ☀️😊 Can you write in here so I can assign! Just a quick note, making issues like this so you can document some of the work you've been doing on Phabricator! 👏

**Comment by mhmohona** [comment]
Thank you. 😃

**Comment by andrewtavis** [comment]
Thanks for this, @mhmohona!

**Comment by andrewtavis** [comment]
Note from #140 where I was thinking about this:

```bash
scribe-data list --language  # list all languages
scribe-data list --word-type  # list all word types
scribe-data list --language German --word-type  # list all German word types
scribe-data list --language --word-type nouns  # list all languages that you can get nouns for
```

Open to how this could improve further!

**Comment by mhmohona** [comment]
@andrewtavis,I have updated the commands - 
![image](https://github.com/scribe-org/Scribe-Data/assets/14244685/13b3543e-ea8b-4558-8d43-5b0e4633b5ad)

**Comment by andrewtavis** [comment]
Closed by #140 🚀 Thank you, @mhmohona! Here's a quick note on what I changed the outputs to, btw:

```bash
(venv) scribe-org/Scribe-Data » scribe-data l

Language     ISO  QID    
-----------------------
English      en   Q1860  
French       fr   Q150   
German       de   Q188   
Italian      it   Q652   
Portuguese   pt   Q5146  
Russian      ru   Q7737  
Spanish      es   Q1321  
Swedish      sv   Q9027  
-----------------------


Available word types: All languages
-----------------------------------
nouns
prepositions
translations
verbs
-----------------------------------
```

Again, we'll need to figure out a way to get `emoji_keywords` and `autosuggestions` back into the word types, which might require us to but a directory for them under each language for which they're implemented :) That'd be ok, I think 😊

First fully functional command! Great work so far!
